bin/functions/load_config.py

```diff
@@ -736,10 +737,10 @@ def export_config(workload_name):
     if len(sys.argv) < 4:
         raise Exception(
             "Please supply <conf root path>, <workload root path>, <workload folder path> [<patch config lists, seperated by comma>")
-    conf_root, workload_config, workload_name = sys.argv[
+    conf_root, workload_configFile, workload_folder = sys.argv[
```

Comment:
The original style seems to be strange here? @gczsjdy, can you check all similar occurrences later in another PR ?
PEP8 limits all lines to a maximum of 79 characters. May be we can make it to 100 characters as an exception, just like Spark's python code.